src/Polymath.ts

```diff
@@ -431,6 +434,35 @@ export class Polymath {
     return await procedure.prepare();
   };
 
+  public cancelSTO = async (args: {
+    moduleUid:
```

Comment:
This should be renamed to `stoModuleId` for consistency, and should only be a string (we could make the case for every UID passed as a parameter to be able to be passed as an object, but then we'd have to do it for every function that accepts a UID as a parameter, including `securityTokenId` in this function, and that might be very messy)